src/how-to/create-dates.md

```diff
@@ -0,0 +1,87 @@
+# Create dates and times
+
+`time` provides a [`const`](https://doc.rust-lang.org/std/keyword.const.html)-ready API. All 
+functions here are `const`: values are calculated at compile time, not at runtime.
```

Comment:
Note that a function being `const` does not necessarily mean that the result _is_ calculated at compile time, just that it _can be_ if needed.